File: app/src/main/java/dev/ragnarok/fenrir/api/model/response/UploadChatPhotoResponse.java
package dev.ragnarok.fenrir.api.model.response;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class UploadChatPhotoResponse {

    @SerializedName("message_id")
    public int message_id;

    @Nullable
    @SerializedName("chat")
    public ChatPhoto chat;

    public static class ChatPhoto {
        @Nullable
        @SerializedName("photo_50")
        public String photo_50;

        @Nullable
        @SerializedName("photo_100")
        public String photo_100;

        @Nullable
        @SerializedName("photo_200")
        public String photo_200;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/response/UploadChatPhotoResponse.kt
package dev.ragnarok.fenrir.api.model.response

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class UploadChatPhotoResponse {
    @SerialName("message_id")
    var message_id = 0

    @SerialName("chat")
    var chat: ChatPhoto? = null

    @Serializable
    class ChatPhoto {
        @SerialName("photo_50")
        var photo_50: String? = null

        @SerialName("photo_100")
        var photo_100: String? = null

        @SerialName("photo_200")
        var photo_200: String? = null
    }
}